### PR TITLE
Black

### PR DESCRIPTION
**Pull request by jimmeak**
Tests in fork went fine. Locally tested only by human - adding new phorum comments.

### CODE REVIEW

File: static/common.css [hunk 1]
```diff
@@ -170,10 +170,29 @@ article.link span.actuallink {
...
+
+p.annotation{
+    font-size: 0.95rem
+}
+
 .comment {
     display: flex;
 }
 
+.Comment--TextArea{
```
Comment (Almad) [inline static/common.css:182]: I just think we should be consistent. So either `.comment--textarea` or rename everything else to be CamelCase.

But this looks like BEM, which I am all for—but [in BEM docs](http://getbem.com/introduction/), it's all lowercase and actually, `--` is for states. 

So IMHO it should be `.comment`, `.comment__textarea` and `.comment_submit`. 

And yeah, we should decide and migrate the rest as well :)

File: ddcz/views.py [hunk 2]
```diff
@@ -302,4 +311,5 @@ def phorum(request):
 
     return render(request, 'discussions/phorum-list.html', {
         'discussions': discussions,
-    })
+        'phorumCommentaryForm': PhorumCommentaryForm()
+    })
```
Comment (Almad) [inline ddcz/views.py:315]: Missing newline—please configure your editor to add it

**Review by Almad — CHANGES_REQUESTED** [review]

File: ddcz/views.py (hunk 2 above)
Comment (jimmeak) [reply ddcz/views.py]: New line added

File: static/common.css (hunk 1 above)
Comment (jimmeak) [reply static/common.css]: Redone by BEM. More in slack.

**Comment by jimmeak** [comment]
Divided into two other pull requests because of branch came with new code and did get black all the code in project.

**Pull request closed without merging**